src/Compilers/VisualBasic/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.vb

```diff
@@ -4273,6 +4273,120 @@ End Class
             diff2.VerifyIL("C.F", expectedIL.Replace("<<VALUE>>", "2"))
         End Sub
 
+        <Fact>
+        Public Sub AnonymousDelegates1()
+            Dim source0 = MarkedSource("
+Class C
+    Private Sub F()
+        Dim <N:0>g</N:0> = <N:1>Function(ByRef arg As String) arg</N:1>
+        System.Console.WriteLine(1)
+    End Sub
+End Class
+")
+            Dim source1 = MarkedSource("
+Class C
+    Private Sub F()
+        Dim <N:0>g</N:0> = <N:1>Function(ByRef arg As String) arg</N:1>
+        System.Console.WriteLine(2)
+    End Sub
+End Class
+")
+            Dim source2 = MarkedSource("
+Class C
+    Private Sub F()
+        Dim <N:0>g</N:0> = <N:1>Function(ByRef arg As String) arg</N:1>
+        System.Console.WriteLine(3)
+    End Sub
+End Class
+")
```

Comment:
Consider adding a test where anonymous delegate signatures match across generations other than ref-ness of parameters.